src/main/java/io/github/christophermanahan/carnitas/Main.java

```diff
@@ -27,4 +27,13 @@ private static int port(String[] args) {
           .map(Integer::parseInt)
           .orElse(33333);
     }
+
+    private static Handler router() {
```

Comment:
I think this should pulled out into a `Configuration` class

Reply:
Yeah, what is this `router`? What is its purpose? It seems to me that it is the application that will help satisfy our acceptance tests.

Reply:
It has been pulled out into an application class that provides a router to satisfy the acceptance tests.